build/test/plugins/.eslintrc.cjs

```diff
@@ -0,0 +1,12 @@
+/**
+ * @license Copyright 2022 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+module.exports = {
+  env: {
+    'jest/globals': true,
+  },
+  plugins: ['jest'],
```

Comment:
wait, what's going on here? #13884 definitely doesn't mention adding more rules :)

Reply:
The plugin is required to use jest/globals.

<img width="787" alt="image" src="https://user-images.githubusercontent.com/4071474/166970209-0a7e9396-b238-4a43-8f5c-5fa4fce9045b.png">

It doesn't enable any additional rules. Only if you extend from their configs. https://www.npmjs.com/package/eslint-plugin-jest#:~:text=Shareable%20configurations

Reply:
right, looks like `jest/globals` is from the plugin, but why do we need `jest/globals`?

Reply:
in https://github.com/GoogleChrome/lighthouse/pull/13295/files#r854447349 you suggested we use `env: 'jest'`. The `env` property actually needs to be an object (it fails config validation otherwise). `jest/globals` is what defines `it`, `describe`, etc on the global scope. Replacing the job of all these comments.

Were you expecting something else?

Reply:
the equivalent of the in-file `/* eslint-env jest */` is `jest: true` in the `env` blocks in the eslintrc files. AFAIK those two should behave identically